rust/connlib/tunnel/src/messages/gateway.rs

```diff
@@ -195,6 +198,7 @@ pub enum IngressMessages {
     RelaysPresence(RelaysPresence),
     ResourceUpdated(ResourceDescription),
     AuthorizeFlow(AuthorizeFlow),
+    AccessAuthorizationExpiryUpdated(AccessAuthorizationExpiryUpdated),
```

Comment:
@jamilbk I wasn't quite sure what you are going to name things here so I just made something up. With the transition of `flow -> authorization` I figured that calling these things an "access authorization" actually sounds quite nice so I went with that. Happy to drop the "access" though if you just want to call them "authorizations".

Reply:
The name of the message here would be `access_authorization_expiry_updated`.